ts/editor/Fields.svelte

```diff
@@ -12,6 +12,7 @@ Contains the fields. This contains the scrollable area.
 
 <style lang="scss">
     .fields {
+        padding-top: 5px;
```

Comment:
<img width="674" alt="Bildschirmfoto 2022-09-13 um 00 57 08" src="https://user-images.githubusercontent.com/7188844/189772888-78188e6c-a5bb-4083-83b0-e6d49259c4f6.png">

This will expose some area above the label container. Should probably be margin-top.
Alternatively, we could drop the grid-gap a few rows below, and add the padding-top in the LabelContainer component.